docs/terraform.md

```diff
@@ -12,6 +12,7 @@
 | Name | Version |
 |------|---------|
 | <a name="provider_aws"></a> [aws](#provider\_aws) | >= 2.0 |
+| <a name="provider_random"></a> [random](#provider\_random) | n/a |
```

Comment:
@dkossako can you please add the random provider explicitly to `versions.tf`?

Reply:
fixed